### DISCUSSION TITLE
Is there vscode-like command palette?

### DISCUSSION THREAD

**Discussion by AdamJel** [wezterm/wezterm#2673] [Q&A] [answered]
Hi,

is there any feature similar to command palette in vscode? I mostly use the command palette to search for commands for which I don't remember the shortcut (or the shortcut is not even set). I really like the overall concept of the command palette and it would be great to have the same feature here.

**Comment by wez** [comment, marked as answer]
* https://github.com/wez/wezterm/issues/1485

**Reply by AdamJel** [reply]
Awesome. Looking forward to it.
